packages/@aws-cdk/toolkit-lib/lib/index.ts

```diff
@@ -7,11 +7,13 @@ import './private/dispose-polyfill';
 
 // The main show
 export * from './toolkit';
+export * from './toolkit/toolkit-error';
```

Comment:
Why like this and not via the previous `./toolkit` export?

Reply:
This one's a bit annoying. Previously `ToolkitError` et.al. where exported via `shared-public.ts`. Adding it to the barrel file `lib/toolkit/index.ts` can too easily cause circular imports, because `ToolkitError` is used all over the place. So its best to import it directly. My idea/hope was that only exporting `ToolkitError` from the top-level public export will discourage a potentially problematic import from the lower level.